### PR TITLE
Use gdrive-datastore

### PR DESCRIPTION
**Pull request by ChenglimEar**
The pypi package gdrive-datastore is now a central place for the code that allows storing our data files on Google Drive.  In this pull request, we switch over to a pip install for this code and remove it from the repository.  Additionally, dev container support has been added to ensure the required development environment is completely specified.

### CODE REVIEW

File: .devcontainer/devcontainer.json
Comment (ckingbailey) [inline .devcontainer/devcontainer.json:1]: 👀  This is a CodeSpaces thing? Nifty. I still need to try it out.

File: test_pull_from_gdrive.py [hunk 1]
```diff
@@ -1,15 +1,4 @@
...
-downloads_dir = '.local/downloads'
-os.makedirs(downloads_dir, exist_ok=True)
-copier = GDriveCopier(GDRIVE_FOLDER, target_branch = REPO_BRANCH)
-copier.download_to(downloads_dir)
-print(f'Contents of downloads dir ({downloads_dir}):')
-local_files = os.listdir(downloads_dir)
-for local_file in local_files:
-    print(local_file)
-print('Done')
+import os
```
Comment (ckingbailey) [inline test_pull_from_gdrive.py]: Unneeded import `os`

**Review by ckingbailey — CHANGES_REQUESTED** [review]
One small thing: unused import

File: test_pull_from_gdrive.py (hunk 1 above)
Comment (ChenglimEar) [reply test_pull_from_gdrive.py]: removed

**Pull request merged**